viewer/packages/camera-manager/src/CameraManager.ts

```diff
@@ -526,25 +529,25 @@ export class CameraManager {
     const onWheel = async (e: any) => {
       const timeDelta = wheelClock.getDelta();
 
+      if (timeDelta > 0.08) scrollStarted = false;
```

Comment:
Magic number :eyes: What does it mean?

Reply:
It's a magic number, yeah. Will extract it in a variable. Basically it is the smallest possible time between 2 distinct model raycasts, i.e. scrollTarget updates